src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimCallbackHandler.java

```diff
@@ -688,6 +692,7 @@ private void assembleResponseDocuments(CaseData caseData, CaseData.CaseDataBuild
         }
     }
 
+    //This line will not be reached when AddApplicant2 Is No, so can never be fully covered
```

Comment:
I'll need to understand how you got to this conclusion.

Reply:
Have added a Positive scenario